docker/run_tests_python3.sh

```diff
@@ -11,6 +11,7 @@
 # This is the list of known working tests by directory/filename. When you
 # have tests in a directory/file working, add it to this list as a new line.
 WORKING_TESTS=(
+    #socorro/unittest/lib/*.py
```

Comment:
Could we remove the commented-out tests? Since they're commented out, they're not being run, and including them in the PR that we merge just adds more lines that aren't really useful for other developers. You can add them back in to your file after you commit and push the changes so that they're still on your laptop to help you keep track.

I can also remove them when merging this. Let me know if I should do that instead. :D